starknet_devnet/fee_token.py

```diff
@@ -131,14 +133,26 @@ async def mint(self, to_address: int, amount: int, lite: bool):
 
         tx_hash = None
         transaction = await self.get_mint_transaction(to_address, amount_uint256)
+        logger.info(transaction)
         starknet: Starknet = self.starknet_wrapper.starknet
         if lite:
             internal_tx = InternalInvokeFunction.from_external(
                 transaction, starknet.state.general_config
             )
-            await starknet.state.execute_tx(internal_tx)
+            execution_info = await starknet.state.execute_tx(internal_tx)
+            logger.info(
+                "transaction execution info: %s", pprint.pformat(execution_info.dump())
```

Comment:
Why log only for lite requests?

Reply:
regular logs in TransactionHandler

Reply:
Then please add a comment line explaining this.